src/template.html

```diff
@@ -14,6 +14,12 @@
   <div class="container">
     <h3>Today's to do:</h3>
     <div id="todo" class="todo"></div>
+          <label for="add-todo" id="add-todo-label">
+            <input type="text" name="add-todo" id="add-todo" placeholder="Add to your list..." />
+            <img id="enter" />
+          </label>
+          <ul id="list"></ul>
+
     <button id="todoBtn" class="btn">Clear all completed</button>
   </div>
```

Comment:
- [ ] The display of tasks should be in a javascript file. This means you should have a method that iterates over the tasks above and render them in HTML.

- [ ] Kindly ensure your styling matches the wireframe from the video.

  Your Implementation | The Design
  :---:|:---:
  ![rueo](https://user-images.githubusercontent.com/81468675/213226881-d90dd29b-144f-4db4-9804-972a10f44b04.jpg) | ![kdkd](https://user-images.githubusercontent.com/81468675/213226896-cefa0999-51f9-4f36-bd29-73d4f73ed647.jpg)